tier4_system_msgs/CMakeLists.txt

```
@@ -21,6 +21,7 @@ rosidl_generate_interfaces(${PROJECT_NAME}
   "msg/HazardStatus.msg"
   "msg/HazardStatusStamped.msg"
   "msg/ServiceLog.msg"
+  "msg/ModeChangeAvailable.msg"
```

Considering there is already `IsAutonomousAvailable`, is adding `Is` better?

```suggestion
  "msg/IsModeChangeAvailable.msg"
```

---

Use `ModeChangeAvailable` as a generalization of `IsModeChangeAvailable`. I'll remove `IsModeChangeAvailable` later.